portal-frontend/src/app/features/public/application/submission/naru-details/naru-details.component.html

```diff
@@ -163,38 +163,40 @@
-        <span *ngIf="applicationSubmission.naruToPlaceVolume !== null">m<sup>3</sup></span>
-        <app-no-data *ngIf="applicationSubmission.naruToPlaceVolume === null"></app-no-data>
-      </div>
-      <div class="grid-1"><strong>Area</strong></div>
-      <div class="grid-2">
-        {{ applicationSubmission.naruToPlaceArea }}
-        <span *ngIf="applicationSubmission.naruToPlaceArea !== null">m<sup>2</sup></span>
-        <app-no-data *ngIf="applicationSubmission.naruToPlaceArea === null"></app-no-data>
-      </div>
-      <div class="grid-1"><strong>Maximum Depth</strong></div>
-      <div class="grid-2">
-        {{ applicationSubmission.naruToPlaceMaximumDepth }}
-        <span *ngIf="applicationSubmission.naruToPlaceMaximumDepth !== null">m</span>
-        <app-no-data *ngIf="applicationSubmission.naruToPlaceMaximumDepth === null"></app-no-data>
-      </div>
-      <div class="grid-1"><strong>Average Depth</strong></div>
-      <div class="grid-2">
-        {{ applicationSubmission.naruToPlaceAverageDepth }}
-        <span *ngIf="applicationSubmission.naruToPlaceAverageDepth !== null">m</span>
-        <app-no-data *ngIf="applicationSubmission.naruToPlaceAverageDepth === null"></app-no-data>
-      </div>
-    </div>
-
-    <div class="full-width">
-      <span class="subheading1">Project Duration</span>
+    <div class="subheading2 grid-1">Volume</div>
+    <div class="grid-double">
+      {{ applicationSubmission.naruToPlaceVolume }}
+      <span *ngIf="applicationSubmission.naruToPlaceVolume !== null">m<sup>3</sup></span>
+      <app-no-data *ngIf="applicationSubmission.naruToPlaceVolume === null"></app-no-data>
+    </div>
+    <div class="subheading2 grid-1">Area</div>
+    <div class="grid-double">
+      {{ applicationSubmission.naruToPlaceArea }}
+      <span *ngIf="applicationSubmission.naruToPlaceArea !== null">m<sup>2</sup></span>
+      <app-no-data *ngIf="applicationSubmission.naruToPlaceArea === null"></app-no-data>
+    </div>
+    <div class="subheading2 grid-1">Maximum Depth</div>
+    <div class="grid-double">
+      {{ applicationSubmission.naruToPlaceMaximumDepth }}
+      <span *ngIf="applicationSubmission.naruToPlaceMaximumDepth !== null">m</span>
+      <app-no-data
+        *ngIf="applicationSubmission.naruToPlaceMaximumDepth === null"
+      ></app-no-data>
+    </div>
+    <div class="subheading2 grid-1">Average Depth</div>
+    <div class="grid-double">
+      {{ applicationSubmission.naruToPlaceAverageDepth }}
+      <span *ngIf="applicationSubmission.naruToPlaceAverageDepth !== null">m</span>
+      <app-no-data
+        *ngIf="applicationSubmission.naruToPlaceAverageDepth === null"
+      ></app-no-data>
     </div>
+
+    <div class="grid-1"></div>
+    <div class="subheading2 grid-double">Project Duration</div>
     <div class="subheading2 grid-1">Duration</div>
     <div class="grid-double">
       {{ applicationSubmission.naruProjectDurationUnit }}
```

Comment:
Project duration section needs to be updated like applicant view (before the soil tables and combined into one field)